internal/builders/generic/README.md

```diff
@@ -42,17 +52,15 @@ jobs:
       digest: ${{ steps.hash.outputs.digest }}
     runs-on: ubuntu-latest
     steps:
-      # Your build steps are here.
+      - name: "build binary-linux-amd64"
+        run: |
+          # Build build binary-linux-amd64 here.
       - name: "generate hash"
         shell: bash
         id: hash
         run: |
           set -euo pipefail
-          DIGEST=$(sha256sum binary-linux-amd64)
-          DIGEST="${DIGEST//'%'/'%25'}"
-          DIGEST="${DIGEST//$'\n'/'%0A'}"
-          DIGEST="${DIGEST//$'\r'/'%0D'}"
-          echo "::set-output name=digest::$DIGEST"
+          echo "::set-output name=digest::$(sha256sum binary-linux-amd64 | base64 -w0)"
```

Comment:
This commend is for below on line 82, the `workflow inputs` is documented as the output of sha256sum: can you also add base64-encoded?

Reply:
Done